docker/net-crd-installer.Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Build the crdinstaller binary
+FROM mcr.microsoft.com/oss/go/microsoft/golang:1.24.4 AS builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY cmd/net-crd-installer/ cmd/net-crd-installer/
+
+ARG TARGETARCH
```

Comment:
what is TARGETARCH?

Reply:
ARG TARGETARCH is a Docker build argument that specifies the target architecture for the binary being built. https://docs.docker.com/reference/dockerfile/#automatic-platform-args-in-the-global-scope